### PR TITLE
Oj-6521: bitbucket cloud support & more jira configuration

### PR DESCRIPTION
**Pull request by eliwind**

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -15,5 +15,6 @@ dateparser = "*"
 jsonstreams = "*"
 psutil = "*"
 python-gitlab = "*"
+numpy = ">= 1.18, < 1.19"
```
Comment (ametzger) [inline Pipfile:18]: this doesn't seem to be used anywhere?

Comment (mattklein) [reply Pipfile:18]: It's this line in `jira_download.py`:
```
    issue_ids_for_threads = numpy.array_split(issue_ids_to_download, num_threads_to_use)
```

Could write it without using numpy, but I thought it was better to let them work out the corner cases. Adding numpy increased the image size from 312 MB to 393 MB, which seems OK.

Comment (ametzger) [reply Pipfile:18]: the code to do that seems straightforward enough I'm not convinced it's worth 79M:
``` python
def chunk(l, n):
    for i in range(0, len(l), n):
        yield l[i:i+n]
```

Comment (mattklein) [reply Pipfile:18]: OK fair enough, I'll add that as a TODO

**Pull request merged**